src/main/java/net/starschema/clouddb/jdbc/Oauth2Bigquery.java

```diff
@@ -86,7 +86,7 @@ public class Oauth2Bigquery {
   /** Application name set on bigquery connection */
   static final String applicationName = "BigQuery JDBC Driver";
 
-  private static final String DRIVE_SCOPE = "https://www.googleapis.com/auth/drive";
+  private static final String DRIVE_SCOPE = "https://www.googleapis.com/auth/drive.readonly";
```

Comment:
Assuming this is fair to change for all uses.
I can't imagine why we would need not a readonly scope.

Reply:
BQ does support [the `CREATE EXTERNAL TABLE` DDL statement ](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#create_external_table_statement) so some might want write access to Drive as well.

Reply:
That said, I think it's fine to make it readonly for now and change it back should someone request it.